splinterd/src/error.rs

```diff
@@ -26,6 +26,7 @@ use crate::daemon::StartError;
 #[derive(Debug)]
 pub enum UserError {
     TransportError(GetTransportError),
+    #[cfg(feature = "metrics")]
```

Comment:
This change does not match the commit message.

Reply:
Added an explanation of the linting fix in commit message